server/server.go

```diff
@@ -284,6 +293,10 @@ func NewServer(ctx context.Context, prof Profile) (*Server, error) {
 	}
 
 	// Middleware
+	e.HTTPErrorHandler = func(err error, c echo.Context) {
+		errorRecorderMiddleware(err, s, c, e)
```

Comment:
Can we call `e.DefaultHTTPErrorHandler` before `errorRecorderMiddleware` so that we do not need to duplicate code of `e.DefaultHTTPErrorHandler`?

Reply:
Now we call `e.DefaultHTTPErrorHandler` inside `errorRecorderMiddleware` to reduce the code.